src/execution/helper.rs

```diff
@@ -52,15 +54,26 @@ pub struct ExecutionHelper {
     pub execute_code_read_iter: IntoIter<Felt252>,
     // Per-contract storage
     pub storage_by_address: ContractStorageMap<DictStorage, PedersenHash>,
+
+    pub secp_hint_processors: HashMap<TypeId, SecpHintProcessor>,
```

Comment:
`TypeId`?? Why? Don't you know the type in advance? I also don't see the point of using a hashmap for this, we can simply have one for K1 and another for R1 no?

Reply:
We only have two handlers, right?